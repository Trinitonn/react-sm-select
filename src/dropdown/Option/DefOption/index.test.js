import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { DefOption } from './index';

describe('DefOption', () => {
  it('should render properly', () => {
    const wrapper = shallow(
      <DefOption
        checked={true}
        option={{ value: 'one', label: 'One' }}
        isSingle={false}
      />
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should render properly in Single mode', () => {
    const wrapper = shallow(
      <DefOption
        checked={false}
        option={{ value: 'one', label: 'One' }}
        isSingle={true}
      />
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
