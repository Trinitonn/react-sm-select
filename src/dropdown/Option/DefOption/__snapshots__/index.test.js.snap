// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefOption should render properly 1`] = `
<div
  className="Option__renderer"
>
  <span
    className="Option__label"
  >
    One
  </span>
  <input
    defaultChecked={true}
    tabIndex="-1"
    type="checkbox"
  />
</div>
`;

exports[`DefOption should render properly in Single mode 1`] = `
<div
  className="Option__renderer"
>
  <span
    className="Option__label"
  >
    One
  </span>
</div>
`;
